src/main/java/org/datadog/jmxfetch/JmxAttribute.java

```diff
@@ -195,12 +198,17 @@ private List<String> getBeanParametersList(
      * Sanitize MBean parameter names and values, i.e. - Rename parameter names conflicting with
      * existing tags - Remove illegal characters
      */
-    private static List<String> sanitizeParameters(List<String> beanParametersList) {
+    private List<String> sanitizeParameters(List<String> beanParametersList) {
         List<String> defaultTagsList = new ArrayList<String>(beanParametersList.size());
         for (String rawBeanParameter : beanParametersList) {
             // Remove `|` characters
             String beanParameter = rawBeanParameter.replace("|", "");
 
+            // Remove `"` characters
+            if(normalizeBeanParamTags == true) {
+                beanParameter = beanParameter.replace("\"","");
```

Comment:
do you think we should remove all quotes or only the quotes at the start/end of the bean parameter?